test/feature/win/parameters.yml

```diff
@@ -0,0 +1,59 @@
+# Receivers that agent needs to tests
```

Comment:
The parameters which is used for generator configuration. However, if you see the validate_type is feature and only be able [after this PR](https://github.com/aws/amazon-cloudwatch-agent-test/pull/162)